bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxPlayerHandler.java

```diff
@@ -289,6 +296,16 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             case CHANNEL_FAVORITES_PLAY:
                 squeezeBoxServerHandler.playFavorite(mac, command.toString());
                 break;
+            case CHANNEL_LIKE:
+                if (likeCommand != null) {
+                    squeezeBoxServerHandler.like(mac, likeCommand);
+                }
```

Comment:
So the `command` param isn't used? Shouldn't you confirm that the `ON` command was sent?

Reply:
Yeah, I thought about that, and I probably should check for ON. OTOH, I was trying to be flexible in allowing any command to trigger the operation. I'll add the check.